Security (Is it safe?)/Data security/Vulnerability disclosure program.yaml

```diff
@@ -9,6 +9,13 @@ criterias:
       - indicator: >+
           The company has a mechanism (ex: a bug bounty program) through which security researchers can
           submit vulnerabilities they discover.
+          
+          A company with a bug bounty program has an alternative mechanism to report vulnerabilities that non-technical 
+          consumers can use to report vulnerabilities.
```

Comment:
Instead of asking conpanies for having a vulnerability report portal for non-technical consumers, is it ok to say that A company with a bug bounty program has an alternative mechanism to report bugs that non-technical consumers can use.